packages/browser-wallet/src/popup/pages/About/About.scss

```diff
@@ -0,0 +1,23 @@
+.about-page {
+    height: 100%;
+    background-color: $color-bg;
```

Comment:
The About page not using the bg picture, is that your artistic vision?

Reply:
I extrapolated it from the settings page.